### PR TITLE
first pass at post processing function

### PR DESCRIPTION
**Pull request by jherzberg**

### CODE REVIEW

**Comment by jherzberg** [comment]
see comments in file

**Comment by kbrose** [comment]
Could we throw this into `lib\tagnews\geoloc\tag.py` where there's currently a stub for the `post_process` function?

Just replacing this function: https://github.com/chicago-justice-project/article-tagging/blob/master/lib/tagnews/geoloc/tag.py#L29-L33

**Comment by kbrose** [comment]
Also, ref: https://github.com/chicago-justice-project/article-tagging/issues/93

File: lib/tagnews/geoloc/tag.py [hunk 1]
```diff
@@ -27,9 +29,18 @@
...
+    assert type(
+        addr_list) == str, "use a string! this is an error from the post_process\
+                            function in tag.py"
+    clean = re.sub('[\W]+ ', '', addr_string)
+    if 'chicago' not in geostring.lower():
+        geostring = geostring + ' Chicago'
+    if ' il' not in geostring.lower() and ' illnois' not in geostring.lower():
+        geostring = geostring + ', Illinois'
+    for word in geostring.split(' '):
+        if word in stopwords.words('english') or word.lower == 'block':
```
Comment (kbrose) [inline lib/tagnews/geoloc/tag.py]: I think there's missing parantheses on `word.lower == 'block'`

**Comment by jherzberg** [comment]
I should probably unit test. @kbrose, parentheses added.

File: lib/tagnews/geoloc/tag.py [hunk 2]
```diff
@@ -27,9 +29,18 @@
 
 
 def post_process(geostring):
-    # TODO
-    geostring += ' Chicago, Illinois'
-    geostring.replace('block of ', '')
+    assert type(
```
Comment (kbrose) [inline lib/tagnews/geoloc/tag.py]: I don't think we need this check anymore, also `addr_list` is not a variable in this function

File: lib/tagnews/geoloc/tag.py [hunk 3]
```diff
@@ -27,9 +29,18 @@
...
+        addr_list) == str, "use a string! this is an error from the post_process\
+                            function in tag.py"
+    clean = re.sub('[\W]+ ', '', addr_string)
```
Comment (kbrose) [inline lib/tagnews/geoloc/tag.py]: `addr_string` doesn't exist and `clean` is assigned but never used, probably want to change both to `geostring`:

```
geostring = re.sub('[\W]+ ', '', geostring)
```

**Comment by kbrose** [comment]
No problem @jherzberg - I'm actually planning on working on getting continuous unit tests running tonight. In the mean time added a few more comments.

**Comment by kbrose** [comment]
Thanks for your hard work @jherzberg !

**Pull request merged**